composer.json

```diff
@@ -30,6 +29,7 @@
         "openeuropa/drupal-core-require-dev": "^8.7",
         "openeuropa/oe_content": "dev-master",
         "openeuropa/oe_corporate_blocks": "~1.0",
+        "openeuropa/oe_media": "dev-OPENEUROPA-1946 as 1.0.0-beta3",
```

Comment:
Let's use dev-master like we did with oe_content (once OPENEUROPA-194 is merged). It's dev dependencies anyway.